sources/packages/backend/libs/sims-db/src/entities/education-program-offering.model.ts

```diff
@@ -158,6 +158,24 @@ export class EducationProgramOffering extends RecordDataModel {
   })
   yearOfStudy: number;
 
+  /**
+   * Indicates if the offering is an aviation offering.
+   */
+  @Column({
+    name: "is_aviation_offering",
+    nullable: false,
+  })
+  isAviationOffering: AviationYesNoOptions;
```

Comment:
For clarification, other similar columns were defined as a string, and the business layer was usually defined as an enum when needed to interact with it. Was there a reason for this one to be different?